influxql/ast.go

```diff
@@ -1296,14 +1296,27 @@ func (s *SelectStatement) String() string {
 }
 
 // RequiredPrivileges returns the privilege required to execute the SelectStatement.
-func (s *SelectStatement) RequiredPrivileges() ExecutionPrivileges {
-	ep := ExecutionPrivileges{{Admin: false, Name: "", Privilege: ReadPrivilege}}
+// NOTE: Statement must be normalized first (database name(s) in Sources and
```

Comment:
In the normal course of things, the statement will not be normalized yet. The comment should talk about how it's OK because the default database gets filled in correctly anyway.